### DISCUSSION TITLE
Welcome to kubernetes Technology Discussions!

### DISCUSSION THREAD

**Discussion by gkdevops** [gkdevops/kubernetes-code#1] [General]
## 👋 Welcome DevOps Folks!
  We’re using Discussions as a place to connect with other members of our Kubernetes learning community. I hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * Maintain your question strictly to the technical aspects only

  To get started, comment below with a question/tip you have on Kubernetes Platform

**Comment by priyanka24-hash** [comment]
Hello,

I created 2 pods with replicaset in deployment.yaml file, which contains image which I created.
I can access the pods service , nodes...everything looks fine.
But when I tried to use curl command as given in screenshot I am getting 403 error. Not sure why..

[kubernetes_issue.docx](https://github.com/gkdevops/kubernetes-code/files/6602384/kubernetes_issue.docx)
